### PR TITLE
Added a mailer to user_controller with tests

### PR DESCRIPTION
**Pull request by Misha7776**
Added a mailer to send credentials. 
Issue #79

### CODE REVIEW

File: spec/mailers/users_mailer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe UsersMailer, type: :mailer do
+  before(:each) do
+    ActionMailer::Base.delivery_method = :test
```
Comment (igorkasyanchuk) [inline spec/mailers/users_mailer_spec.rb]: remove these 3 lines
TEST NEVER SENDING EMAIL! NEVER
in test.rb leave default configuration

File: spec/mailers/users_mailer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    it 'should set the subject to the correct subject' do
+      ActionMailer::Base.deliveries.first.subject.should == 'Your credentials'
+    end
+
+    it 'renders the sender email' do
+      ActionMailer::Base.deliveries.first.from.should == ['salesassistant@mail.com']
+    end
+  end
+  describe 'credentials views' do
+    let(:mail) { UsersMailer.credentials(@user) }
```
Comment (igorkasyanchuk) [inline spec/mailers/users_mailer_spec.rb]: ALWAYS use "let!"

File: spec/mailers/users_mailer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    @user = FactoryBot.create(:user, :sale)
+    UsersMailer.credentials(@user).deliver
+  end
+
+  after(:each) do
+    ActionMailer::Base.deliveries.clear
+  end
+
+  describe 'credentials' do
+    it 'should send an email' do
```
Comment (igorkasyanchuk) [inline spec/mailers/users_mailer_spec.rb]: do not use ActionMailer::Base.deliveries.first or anything like that

rewrite specs like here
https://relishapp.com/rspec/rspec-rails/docs/mailer-specs

but use "let!"

File: app/views/users_mailer/credentials.html.slim [hunk 4]
```diff
@@ -0,0 +1,12 @@
+div
+  h1 Your credentials
+div
+  h3
+    = "#{@greeting}, #{@user.full_name}."
```
Comment (igorkasyanchuk) [inline app/views/users_mailer/credentials.html.slim]: why you have here @greeting? Just put Hello in template
never repeat this error

**Comment by Misha7776** [comment]
app/controllers/account/users_controller.rb -- 1 warning:
  [28]:TooManyStatements: Account::UsersController#create has approx 6 statements [https://github.com/troessner/reek/blob/master/docs/Too-Many-Statements.md]
1 total warning 
----
and 
----
2.82s$ bundle exec rails_best_practices -e "db/migrate,vendor" .
Source Code: |=================================================================|
/home/travis/build/Ruby-IF081/heroes_team_app/app/mailers/users_mailer.rb:2 - use multipart/alternative as content_type of email

**Pull request merged**